### PR TITLE
Add missing PICS to TC-SWTCH tests

### LINKED ISSUE #35358: [TC-SWTCH-##] High level PICS has to be added in the Python script.
**Description:**

- [ ] - In the following switch test case, high-level PICS need to be added according to the test plan.

1. TC-SWTCH-2.2
2. TC-SWTCH-2.3
3. TC-SWTCH-2.4
4. TC-SWTCH-2.5
5. TC-SWTCH-2.6

- [x] - TC_SWTCH_2_4 - In Test step 5  ` if MSL or AS feature is supported, TH expect receiving LongPress/LongRelease in that order` has to be updated to ` if MSL feature is supported, TH expect receiving LongPress/LongRelease in that order` as per the Test plan.

- [x] - TC_SWTCH_2_6 - Test step 9b  `TH receives MultiPressComplete event with PreviousPosition set to 1 and TotalNumberOfPressesCounted set to 2 from the DUT"` has to be updated to `TH receives MultiPressComplete event with PreviousPosition set to 1 and TotalNumberOfPressesCounted set to 1 from the DUT` as per the Test plan.

**Test plan References:**
- https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/switch.adoc

**Python Script References:**
- https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_SWTCH.py

### GitHub hash of the SDK that was being used

708558881bf6e5b0dac3c09489124907927b6cd2

### PR DESCRIPTION
**Pull request by tcarmelveilleux**
Fixes #35358

### CODE REVIEW

**Review by andy31415 — DISMISSED** [review]

File: src/python_testing/TC_SWTCH.py [hunk 1]
```diff
@@ -835,6 +859,13 @@ def steps_TC_SWTCH_2_6(self):
                          "Verify that the TH receives MultiPressComplete event with PreviousPosition set to 1 and TotalNumberOfPressesCounted set to 1 from the DUT"),
                 ]
 
+    def pics_TC_SWTCH_2_6(self) -> list[str]:
```
Comment (andy31415) [inline src/python_testing/TC_SWTCH.py:862]: apparently we double-check for run_if_endpoint_matches for these tests and those cannot have pics (we check matching conditions, not pics here)

Comment (tcarmelveilleux) [reply src/python_testing/TC_SWTCH.py:862]: That is then a bug...

Comment (cecille) [reply src/python_testing/TC_SWTCH.py:862]: no, it's not - these tests are meant to be selected unconditionally, so they have a blank PICS (ie, they're not gated on any condition). TH selects these always, then the test reports if it is applicable.

Comment (cecille) [reply src/python_testing/TC_SWTCH.py:862]: run if endpoint matches is just a slightly different mechanism for gating on the PICS.

Comment (tcarmelveilleux) [reply src/python_testing/TC_SWTCH.py:862]: @cecille said it's not needed.

**Pull request closed without merging**